lib/glimesh/accounts.ex

```diff
@@ -534,5 +550,40 @@ defmodule Glimesh.Accounts do
 
   def can_stream?(user), do: user.can_stream
 
-  def can_use_payments?(user), do: user.can_payments
+  def can_use_payments?(user) do
+    user.can_payments
+  end
+
+  def ban_user(user, reason) do
+    case reason do
```

Comment:
These unsafe functions should be locked behind a permission check here instead of in the controller. This helps ensure we're always writing safe code. An example can be found here: https://github.com/Glimesh/glimesh.tv/blob/dev/lib/glimesh/stream_moderation.ex#L122